manifests/init.pp

```diff
@@ -95,17 +95,23 @@
   Simplib::Netlist              $trusted_nets   = ['127.0.0.1', '::1'],
   String                        $package_ensure = 'latest',
   Boolean                       $libkv          = false,
+  Boolean                       $puppet         = true,
```

Comment:
This should probably default to whether or not the `puppetserver` is loaded on the local system.

Reply:
@lnemsick-simp It looks like we can probably create a fact (something like `simplib_puppetserver_installed`?) that checks to see if both `/etc/puppetlabs/puppetserver/conf.d/global.conf` exists and `/opt/puppetlabs/server/bin/` has items in it (may be symlinks).

The first would see if we're trying to be configured as a server and the second would take care of the case that the application was removed but the config files remained on the system.

It's possible that just the second is enough.

Ref: https://puppet.com/docs/puppetserver/6.5/configuration.html

Reply:
Just because the RPM is installed doesn't mean the puppetserver service is enabled. I don't think checking for the existence of files is sufficient.

Reply:
The service doesn't need to be enabled. We just have to know that we're on a puppetserver. It may (will probably) be enabled by `pupmod` later.

Alternatively, we just don't make any assumptions any longer which would mean defaulting this to `false` and prompting the user in `simp config`.

Reply:
Defaulting to `false` is not a backward compatible change.

Reply:
Yup, thus the general whack at trying to figure out if we're on a puppetserver. At least then it's "best guess" backward compatible and won't surprise anyone when running under Bolt.